src/instruments/ecam-page-template/rollup.js

```diff
@@ -0,0 +1,60 @@
+/*
```

Comment:
This file might look different or no longer be neceesary after the updated rollup.config from @theomessin has landed, not quire sure.

Reply:
That probably won't land for a while.